test/Microsoft.DotNet.DarcLib.Tests/Helpers/DependencyFileManagerTests.cs

```diff
@@ -361,4 +362,27 @@ await manager.AddDependencyAsync(
         versionDetails.Replace("\r\n", "\n").TrimEnd().Should().Be(expectedVersionDetails.Replace("\r\n", "\n").TrimEnd());
         versionProps.Replace("\r\n", "\n").TrimEnd().Should().Be(expectedVersionProps.Replace("\r\n", "\n").TrimEnd());
     }
+
+    [Test]
+    public void GetXmlDocumentHandlesBomCharacters()
+    {
+        // Create XML content without BOM
+        const string xmlWithoutBom =
+            """
+            <?xml version="1.0" encoding="utf-8"?>
+            <Dependencies>
+              <ProductDependencies>
+                <Dependency Name="TestPackage" Version="1.0.0">
+                  <Uri>https://github.com/test/test</Uri>
+                  <Sha>abc123</Sha>
+                </Dependency>
+              </ProductDependencies>
+            </Dependencies>
+            """;
+
+        // Test BOM representation mentioned in issue (∩╗┐)
```

Comment:
nit: what issue? Probably is not even needed since the name of the test + the actual line should be enough context imo

Reply:
fair point, didn't notice that, thanks